packages/plugin-list/test/Checklist.test.ts

```diff
@@ -2624,61 +2709,155 @@ describePlugin(List, testEditor => {
                             });
                         });
                         describe('With attributes', () => {
-                            it('should add two list items at the end of a checklist with a class', async () => {
-                                await testEditor(BasicEditor, {
-                                    contentBefore:
-                                        '<ul class="checklist a"><li class="checked">abc[]</li></ul>',
-                                    stepFunction: async (editor: JWEditor) => {
-                                        await insertParagraphBreak(editor);
-                                        await insertText(editor, 'b');
-                                        await insertParagraphBreak(editor);
-                                    },
-                                    contentAfter:
-                                        '<ul class="checklist a"><li class="checked">abc</li><li class="unchecked">b</li><li class="unchecked">[]<br></li></ul>',
+                            describe('after unchecked item', () => {
+                                // TODO: do not clone the `IsChecked` modifier
```

Comment:
I thought this TODO would only be present in the skipped tests, not in the duplicated ones ? Did I miss something ?